app/presenters/content_row_presenter.rb

```diff
@@ -26,6 +26,8 @@ def format_satisfaction_percentage(score)
   end
 
   def format_satisfaction_responses(yes_responses, no_responses)
+    return 'No data' if yes_responses.nil? || no_responses.nil?
+
```

Comment:
Possibly stupid question... but if 1 person has responded, or all responses are the same should we really be returning no data?

Reply:
Yeah so it be very weird to for that backend to return say `1` yes_responses and `nil` no_responses. But in that case because we still have a `nil` and we can't calculate the percentage.

In this context `nil` is not the same as 0, `nil` means we don't know the value i.e. it could be 0 or it could 1000 etc. And without knowing exactly the number of both yes and no response we can't calculate the percentage.